lib/item.rb

```diff
@@ -18,7 +18,7 @@ def initialize(input, repository)
     @unit_price = input[:unit_price]
     @created_at = input[:created_at]
     @updated_at = input[:updated_at]
-    @merchant_id = input[:merchant_id]
+    @merchant_id = input[:merchant_id].to_i
```

Comment:
Thank you for updating this! We kept getting nil errors because it wasn't recognizing the merchant id passed in since it was passed in as a string. Super helpful!

Reply:
This made everything click

Reply:
Thank you @klatour324 !